### ISSUE TITLE
Use Monty's renders for Goose X

### ISSUE THREAD

**Issue by dhillondeep** [teamwaterloop/waterloop-website#63]

**Comment by nikolaevra** [comment]
Where specifically? on the flock page? I believe they are already using monty's stuff. I think we should just get rid of goose X alltogether, it literally means nothing because its like 7 generations away. WDYT?

**Comment by dhillondeep** [comment]
Yeah that is not part of our flock. We can make another page about our future and discuss Goose X there maybe
